crates/runtime/src/dataconnector/github.rs

```diff
@@ -47,7 +47,8 @@ pub trait GithubTableArgs: Send + Sync {
     ///   1. The GraphQL query string
     ///   2. The JSON pointer to the data in the response
     ///   3. The depth to unnest the data
-    fn get_graphql_values(&self) -> (GraphQLQuery, JSONPointer, UnnestDepth);
+    ///   4. The GraphQL schema of the response data
+    fn get_graphql_values(&self) -> (GraphQLQuery, JSONPointer, UnnestDepth, Option<SchemaRef>);
```

Comment:
this is turning into quite a long tuple type. maybe we should think about creating a custom return struct/type?

Reply:
Yeah 👍, plan to improve this when adding schema for the next GH connector table